grafana/003-new-files.patch

```diff
@@ -853,15 +937,20 @@ index 000000000..85f0dd6aa
 +  end
 +end}
 +
-+# Unbundle grafana vendor sources and instead use BuildRequires
++# Unbundle Grafana vendor sources and instead use BuildRequires
 +# only on platforms that have enough golang devel support.
 +%if 0%{?rhel} == 0
 +%global           unbundle_vendor_sources 1
 +%endif
 +
++%if 0%{?fedora} >= 30
++# Use vendor sources until both Fedora and Grafana properly support golang modules
++%global           unbundle_vendor_sources 0
++%endif
++
 +
 +Name:             grafana
-+Version:          6.2.2
++Version:          6.3.6
```

Comment:
should this be 6.4.3?

Reply:
No. Actually it does not matter. This piece of patch changes an internal, not used by us, .spec file, not our spec file.